skore/src/skore/item/cross_validate_item.py

```diff
@@ -0,0 +1,175 @@
+"""CrossValidationItem class.
+
+This class represents the output of a cross-validation workflow.
+"""
+
+from __future__ import annotations
+
+import hashlib
+from typing import TYPE_CHECKING, Any
+
+from skore.item.item import Item
+
+if TYPE_CHECKING:
+    import sklearn.base
+
+
+def plot_cross_validation(cv_results: dict):
+    """Plot the result of a cross-validation run."""
+    import altair
+    import pandas
+
+    # df = pandas.DataFrame(cv_results).melt(var_name="metric", value_name="score")
```

Comment:
Just wondering if this is a comment meant to stay, or something you wanted to remove.

Reply:
nice catch